### ISSUE TITLE
Broken formatting in recent blog post about NWU

### ISSUE THREAD

**Issue by gvwilson** [swcarpentry/website-archive#645]
The footnotes at the bottom of <https://software-carpentry.org/blog/2017/01/first-african-partner.html> aren't rendering properly, and the PNG image isn't being displayed.

**Comment by rgaiacs** [comment]
This blog post is a cross-post with Data Carpentry. 

I sent a pull request to fix it on Data Carpentry but I and @ErinBecker [got to a point where we didn't know what was the correct fix](https://github.com/datacarpentry/datacarpentry.github.io/pull/198#issuecomment-274206181) so we had to wait for @anelda. On this mess that I created last Friday I had a pull request to Software Carpentry website to fix it, https://github.com/swcarpentry/website/pull/644/files.
